services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/PublicHankeController.kt

```diff
@@ -2,16 +2,40 @@ package fi.hel.haitaton.hanke
 
 import fi.hel.haitaton.hanke.domain.PublicHanke
 import fi.hel.haitaton.hanke.domain.hankeToPublic
-import org.springframework.beans.factory.annotation.Autowired
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @RequestMapping("/public-hankkeet")
-class PublicHankeController(
-    @Autowired private val hankeService: HankeService,
-) {
-    @GetMapping
+class PublicHankeController(private val hankeService: HankeService) {
+
+    @GetMapping(produces = [APPLICATION_JSON_VALUE])
+    @Operation(summary = "Get list of public hanke")
```

Comment:
Having a description of the concept of public hanke wouldn't hurt here.

Reply:
Done.